Lib/ast.py

```diff
@@ -620,6 +620,15 @@ class AugStore(expr_context):
 class Param(expr_context):
     """Deprecated AST node class.  Unused in Python 3."""
 
+_Unparser = None
+
+def unparse(ast_obj):
+    global _Unparser
+    if _Unparser is None:
+        from _ast_unparse import Unparser as _Unparser
+    unparser = _Unparser()
+    return unparser.visit(ast_obj)
```

Comment:
This avoids `_Unparser = None`, and should be just as fast due to zero-cost `try`
```suggestion
    try:
        unparser = _Unparser()
    except NameError:
        global _Unparser
        from _ast_unparse import Unparser as _Unparser
    return unparser.visit(ast_obj)
```

Reply:
Good point, will push manually as your version is missing a line.